### DISCUSSION TITLE
Microsoft Surface devices - Which version of Microsoft .NET 5 runtime?

### DISCUSSION THREAD

**Discussion by timjobutler** [AntonyCorbett/OnlyM#404] [Q&A] [answered]
Hi @AntonyCorbett,

Several brothers in our cong have Microsoft Surface devices which have ARM-based processors. When downloading and installing the **Microsoft .NET 5.0 Desktop runtime**, should they choose the **Desktop Arm64** installer?

Thanks Antony

Your brother,
Tim

**Comment by AntonyCorbett** [comment, marked as answer]
@timjobutler It's the x86 Desktop installer - we are still running in emulation on Arm devices.

**Reply by timjobutler** [reply]
Excellent, thanks for the clarification Antony

Happy to have this item closed
